apps/web/migrations/0002_team.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you delete this migration file and try creating migrations using this command with name
`python manage.py migrate --settings=settings.dev --name=added_team_model`